app/preact/src/server/framework-preset-preact.ts

```diff
@@ -9,3 +11,19 @@ export function babelDefault(config: TransformOptions) {
     ],
   };
 }
+
+export function webpackFinal(config: Configuration) {
+  return {
+    ...config,
+    resolve: {
+      ...config.resolve,
+      modules: [path.resolve('node_modules'), ...config.resolve.modules],
+      alias: {
+        ...config.resolve.alias,
+        react: require.resolve('preact/compat'),
+        'react-dom/test-utils': require.resolve('preact/test-utils'),
+        'react-dom': require.resolve('preact/compat'),
```

Comment:
See https://github.com/storybookjs/storybook/pull/12403#discussion_r503573637

Reply:
OK, didn't know why that was needed, so I decided to try it out with the kitchen sink app and it seemed to work fine, but I'll do the same trick to make it compatible.